backend/src/app.ts

```diff
@@ -75,16 +80,20 @@ class App {
   // Generates routes and initializes Swagger documentation.
   private generateRoutesAndInitializeSwagger(): void {
     // register swagger route
-    this.express.use("/docs", swaggerUi.serve, async (_req: Request, res: Response) => {
-      return res.send(
-        swaggerUi.generateHTML(await import("../build/swagger.json"))
-      );
-    });
+    this.express.use(
+      "/docs",
+      swaggerUi.serve,
+      async (_req: Request, res: Response) => {
+        return res.send(
+          swaggerUi.generateHTML(await import("../build/swagger.json"))
+        );
+      }
+    );
```

Comment:
btw, should we use some common setup for formatting? @Corgam @guttenberger

Reply:
Created #88.